web/modules/custom/registration_form_custom/registration_form_custom.info.yml

```diff
@@ -0,0 +1,7 @@
+name: Registration Form Custom
+type: module
+description: 'Custom registration form'
+core_version_requirement: ^9 || ^10
+package: Custom
+dependencies:
```

Comment:
I saw the use of weather module in this module, but it isn't in the dependencies